lib/components/primitive-components/Trace/Trace.ts

```diff
@@ -660,17 +662,33 @@ export class Trace
       .list()
       .find((label) => label.source_net_id === toPort.source_port_id)
 
-    if (this.props.schDisplayLabel) {
-      if (
-        (existingFromNetLabel &&
-          existingFromNetLabel.text !== this.props.schDisplayLabel) ||
-        (existingToNetLabel &&
-          existingToNetLabel?.text !== this.props.schDisplayLabel)
-      ) {
-        throw new Error(
-          `Cannot create net label for port ${existingFromNetLabel ? fromPortName : toPortName} because it already has a net label with text "${existingFromNetLabel ? existingFromNetLabel.text : existingToNetLabel?.text}".`,
-        )
-      }
+    if (
+      (existingFromNetLabel &&
+        existingFromNetLabel.text !== this.props.schDisplayLabel) ||
+      (existingToNetLabel &&
+        existingToNetLabel?.text !== this.props.schDisplayLabel)
+    ) {
+      throw new Error(
+        `Cannot create net label for port ${existingFromNetLabel ? fromPortName : toPortName} because it already has a net label with text "${existingFromNetLabel ? existingFromNetLabel.text : existingToNetLabel?.text}".`,
+      )
+    }
+
+    if (
+      this.props.schDisplayLabel?.toLocaleLowerCase().includes("gnd") ||
+      this.props.schDisplayLabel?.toLocaleLowerCase().includes("ground")
+    ) {
+      createDownwardNetLabelSymbol(
+        {
+          fromPort,
+          toPort,
+          fromAnchorPos,
+          toAnchorPos,
+          schDisplayLabel: this.props.schDisplayLabel!,
+          source_trace_id: this.source_trace_id!,
+        },
+        db,
```

Comment:
The context has to be a named object

```suggestion
        { db },
```